README.md

```diff
@@ -350,6 +350,14 @@ _Note that prior to release 1.2 of VexCL the `VEX_FUNCTION` macro had different
 interface. That version is considered deprecated but is still available as
 `VEX_FUNCTION_V1`._
 
+Type casting a vector can be done by creating a user defined function. This has the advantage of beeing backend independent. 
+~~~{.cpp}
+VEX_FUNCTION(float, make_float, (int, i),
+    return (float)i;
+    );
+b = make_float(a) / 2;
+~~~
+The ~~~{.cpp}vex::convert_XXX~~~ are OpenCL binding and cannot be used with CUDA. 
```

Comment:
Inline code should be enclosed with backticks. Triple tildas [result](https://github.com/borissmidt/vexcl/blob/86fa9d8f34df141b9a62428ecd349d50ce2af945/README.md#user-defined-functions) in striked out text.

Also, I would rephrase the last sentence to read
```
Another option would be to use builtin OpenCL functions [convert_*](https://www.khronos.org/registry/cl/sdk/1.1/docs/man/xhtml/convert_T.html)',
but those are obviously only available for OpenCL backend.
```